packages/ethers5/src/client.ts

```diff
@@ -467,7 +467,9 @@ export class Web3Modal extends Web3ModalScaffold {
       EthersStoreUtil.setProviderType('walletConnect')
       EthersStoreUtil.setProvider(WalletConnectProvider as unknown as Provider)
       EthersStoreUtil.setIsConnected(true)
-      EthersStoreUtil.setAddress(WalletConnectProvider.accounts[0] as Address)
+      const address = WalletConnectProvider.accounts?.[0]
```

Comment:
Looks like we are doing identical things in ethers and ethers5 packages, can all of this logic be abstracted under ethers scaffold package and not repeated? Also cc @svenvoskamp here for further feedback on ethers.

Reply:
I added some abstraction on each client but we can't share that logic because each of our ethers package need a different utils function from the OG ethers package.